components/organisms/ProtocolMarket.tsx

```diff
@@ -43,3 +49,10 @@ export const ProtocolTokensMarket = () => {
         <TokenRowMarket address={"0x..."} name={"USD Coin"} volumen={"5.57"} borrowAPY={"3.26"} depositAPY={"2.19"} rewardAaveAPY={"0.96"} />
     </View>;
 };
+const styles = StyleSheet.create({
+    divider: {
+        height: 0.5,
+        width: 275,
```

Comment:
I don't think we have to use a fixed value here. Can we just remove it? If we can't, then can we just use `100%` here?